server.go

```diff
@@ -513,16 +533,16 @@ func (p *sshFxpSetstatPacket) respond(svr *Server) responsePacket {
 
 	fs, err := p.unmarshalFileStat(p.Flags)
 
-	if err == nil && (p.Flags & sshFileXferAttrSize) != 0 {
+	if err == nil && (p.Flags&sshFileXferAttrSize) != 0 {
```

Comment:
The changes here should be a result of a `go fmt` and ¿should have been `go fmt`ed before?

Just checking to make sure that this is consistent with `go fmt` of the most recent version.

Reply:
This is indeed the result of `go fmt`, my local version is `go1.21.1`.

Reply:
I’ve checked with `go1.22.2` it does the same. 👍